frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/topology/pipelinesTopology.cy.ts

```diff
@@ -238,6 +238,7 @@ describe('Pipeline topology', () => {
       taskDrawer.findArgumentCodeBlock().should('not.be.empty');
       taskDrawer.findTaskImage().should('have.text', 'Imagequay.io/hukhan/iris-base:1');
       taskDrawer.findCloseDrawerButton().click();
+
```

Comment:
No changes in this file but white space. You can omit it from the change set now.

Reply:
<img width="1728" alt="Screenshot 2025-05-30 at 15 38 59" src="https://github.com/user-attachments/assets/3f63ce84-9f88-462c-ab49-cc964a3c9216" />
@jenny-s51 to help keep history clean, the UI might be able to help us.